### PR TITLE
Update opensearch build name in releases

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Update opensearch build name in releases
  
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by VijayanB — APPROVED** [review]

**Comment by Shivamdhar** [comment]
Can we please add the results for this command -

```
./gradlew assemble --no-daemon --refresh-dependencies -DskipTests=true
```

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,19 @@ validateNebulaPom.enabled = false
 buildscript {
     ext {
         opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
```
Comment (peterzhuamazon) [inline build.gradle:48]: Reminder that this needs to be `2.2.0-SNAPSHOT` for your 2.2 branch backport.

**Comment by junqiu-lei** [comment]
@Shivamdhar Sure
```
(base) ➜  geospatial-junqiu git:(update_in_gradle) ./gradlew assemble --no-daemon --refresh-dependencies -DskipTests=true
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.4.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 7.4.1
  OS Info               : Mac OS X 12.3.1 (x86_64)
  JDK Version           : 11 (Eclipse Temurin JDK)
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home
  Random Testing Seed   : F060900B9EFB1F9A
  In FIPS 140 mode      : false
=======================================

BUILD SUCCESSFUL in 39s
13 actionable tasks: 12 executed, 1 up-to-date

```

**Review by peterzhuamazon — COMMENTED** [review]
Overall great, just in case you forget:

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**